README.md

```diff
@@ -34,3 +34,30 @@ Use `npm run verify` after modifying the resources or metadata file to ensure th
 - `resourcePath` specifies the path to the content of the resource. It should be relative to the `resources` directory in the root of this repository.
 
 In general, this format corresponds to the `Resource` struct from [adblock-rust](https://github.com/brave/adblock-rust), but the base64-encoded `content` field is replaced by a path to a file for better maintainability. This library will translate the `resourcePath` field back to a base64-encoded `content` field when ready for use.
+
+## Filter list description format
+
+The `filter_lists/*.json` files are lists of elements, each describing a filter list. The format for each element is:
+
+```json
+{
+    "uuid": "49254a7e-396e-4d1f-af48-f5730e22e1ce",
+    "url": "https://raw.githubusercontent.com/brave/adblock-lists/master/brave-lists/new-list.txt",
+    "title": "New Filter Rules",
+    "format": "Standard",
+    "include_redirect_urls": false,
+    "support_url": "https://github.com/brave/adblock-lists"
+}
+```
+
+- `uuid` is a unique identifier used by the browser to identify a filter list, should be generated.
+
+- `url` is the URL where the filter list is actually located and can be downloaded from. Filter list should be a list of rules in the format specified in `format` in a `.txt` file.
+
+- `title` is a human-readable title for the filter list.
+
+- `format` is either ABP/uBO-style format ("Standard", most common) or IP address and hostname ("Hosts").
+
+- `include_redirect_urls` permits parsing of `redirect-url` filter option. This has security implications and should usually be `false`.
```

Comment:
Would be good to mention that `include_redirect_urls` is optional and defaults to `false`